src/layouts/Home/components/Patterns.tsx

```diff
@@ -18,13 +18,11 @@ import { HomeLink } from './HomeLink';
 export const Patterns: React.FC = () => {
   const { patternsImage } = useStaticQuery(
     graphql`
-      query {
+      {
         patternsImage: abstractAsset(layerName: { eq: "home-pillars-patterns" }) {
           childFile {
             childImageSharp {
-              fluid(maxWidth: 808) {
-                ...GatsbyImageSharpFluid
-              }
+              gatsbyImageData(width: 808, layout: CONSTRAINED)
```

Comment:
Since `CONSTRAINED` is the default, does it need to be specified?

Reply:
It doesn't and has been removed.